test/integration/component/test_configdrive.py

```diff
@@ -45,21 +49,17 @@
                              VirtualMachine,
                              VPC,
                              VpcOffering,
-                             Hypervisor)
+                             Hypervisor, Template)
 from marvin.lib.common import (get_domain,
                                get_template,
-                               get_zone,
-                               list_templates)
+                               get_zone)
 from marvin.lib.utils import random_gen
 # Import System Modules
 from nose.plugins.attrib import attr
 from retry import retry
```

Comment:
this dependency is not in the required list in marvin installation

Reply:
`retries` was added as as install dependency in tools/marvin/setup.py